INSTALL_MAC.md

```diff
@@ -0,0 +1,61 @@
+# Installation instructions for Mac
+
+## Usage
+
+### Install prerequisites
+
+The cross compilation toolchain and docker have to be installed. 
+The following instruction have been tested on Mac OS Mojave (10.14).
+
+```bash
+# Ensure your brew install is healthy
+brew doctor
+brew install cmake git wget curl bash-completion qemu
+python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
+```
+
+[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/)
+
+### Building a workspace
+
+```bash
+# Install plugin
+python3 -m pip install --editable src/Colcon-cc-build/colcon_cc_build --user
+
+# Launch a build
+## 1. Setup the sysroot
+## add --force-sysroot-build to force rebuilding the sysroot
+colcon cc-setup-sysroot --arch generic_armhf --os ubuntu_bionic \
+   --sysroot-base-image juanrh/ros2:armhf_bionic_crystal_fastrtps_prebuilt-crystal
+## 2. Complete cc-build setup following the instructions in the output of the previous command
+bash generic_armhf-ubuntu_bionic-fastrtps-crystal/cc_system_setup.bash
+source generic_armhf-ubuntu_bionic-fastrtps-crystal/cc_build_setup.bash
+## 3. Launch cross compilation using that sysroot
+colcon cc-build --arch generic_armhf --os ubuntu_bionic \
+  --packages-up-to examples_rclcpp_minimal_publisher
+```
+
+### Assumptions
+
+- The Docker image for `--sysroot-base-image` installs the ROS 2 distro at `/opt/ros/${distro}`.
+
+Additional base images can be created using the script `colcon_cc_build/colcon_cc_build/verb/sysroot/publish_to_ECR.sh`.
+Some base images already available to use with `--sysroot-base-image`:
+
+- `juanrh/ros2:armhf_bionic_crystal_fastrtps_prebuilt-crystal` for building ROS 2 packages starting from a prebuilt binary for ROS 2 crystal for armhf.
```

Comment:
Let's add a FIXME for all those mentions to juanrh to avoid forgeting about it.

Reply:
Fixed in PR #1